syntax = "proto3";

package jio.brain.proto.knowledge.api.data;

option java_package = "com.jio.brain.proto.knowledge.api.data";
option java_multiple_files = true;

import "jio/brain/proto/base/status.proto";

service GetAttributeValueIdService {
    rpc serve(GetAttributeValueIdRequest) returns (GetAttributeValueIdResponse) {}
  }

message GetAttributeValueIdRequest {
    string attribute_value_name = 1;
}

message GetAttributeValueIdResponse {
	uint32 parent_node_id = 1;
    uint32 attribute_id = 2;
    uint32 value_id = 3;
    jio.brain.proto.base.BrainStatus status = 4;
}